client/locations-list-map.js

```diff
@@ -5,7 +5,7 @@ import MarkerClusterer from '@google/markerclustererplus';
 import toDataByLocation from './toDataByLocation.js';
 import countries from './countries.js';
 import { FILTER_ITEMS, ORG_TYPES, ENUM_MAPPINGS } from './formEnumLookups.js';
-import { getCountry, getFirstPathPart, isCountryPath } from './getCountry.js';
+import { getCountry } from './getCountry.js';
```

Comment:
Is this even needed? Can the script just trust body.dataset.countryCode?

Reply:
Just one call to getCountry when setting the map's initial view. I think that can use gCountryCode too.

Reply:
Good point. Used gCountryCode.  I think getCountry() probably should be removed from everywhere now. Leaving for another CL.